test/fixtures/api/responses/get-balance-sheet.json

```diff
@@ -0,0 +1,68 @@
+{
+  "balances": [
+    {
+      "counterparty": "rKm4uWpg9tfwbVSeATv4KxDe6mpE9yPkgJ",
+      "balances": [
```

Comment:
patternProperties will work with `required`? Seems simpler to write schema for non-variable property names